### PR TITLE
  #36 Feature: register driver & operator

### PR DESCRIPTION
**Pull request by rachelakimana**
#### What does this PR do?

- Create register page

#### Description of Task to be completed? 

- Allow an administrator to register a new driver or a new operator.

- Form validation and display appropriate error messages on unsuccessful attempts

- Responsiveness of the

#### How should this be manually tested?

- Clone the repo

- checkout to branch ``ft-register-36``

- Run `npm install` to install modules

- Run `npm  run start` to start the development server

- On the landing page click on the Sign up menu

#### What are the relevant Pivotal Tracker/trello stories?

- https://trello.com/c/Sd4qDC5s/36-register-drivers-operators

#### Questions:

- N/A

### CODE REVIEW

File: src/component/__test__/RegisterPage.test.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+  // it("Test form submit and validation", async () => {
+  //   const handleSubmit = jest.fn();
+  //   render(<RegisterPage onSubmit={handleSubmit} />);
+
+  //   const user = userEvent.setup();
+
+  //   await user.type(screen.getByTestId("email-input"), "rachel@blbla.com");
+  //   await user.type(screen.getByTestId("username-input"), "Rachel");
+  //   await user.type(screen.getByTestId("password-input"), "N#@!Pass");
```
Comment (PrinceNiyonshuti) [inline src/component/__test__/RegisterPage.test.js]: remember to remove the commented codes which are no longer in use

**Review by PrinceNiyonshuti — CHANGES_REQUESTED** [review]
looks good and as additional remember to provide the expected results as it is in frontend and resolve the conflicts that have been occured

**Review by tysepa — APPROVED** [review]
Please work on that collection @PrinceNiyonshuti has been telling you to collect but you working good

**Review by thepatrickniyo — COMMENTED** [review]
Resolve the conflicts as well

**Comment by rachelakimana** [comment]
@PrinceNiyonshuti @PatrickNiyogitare28 
you can review my PR i have done with the requested changes

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]
I'm learning it and get weback compilation errors.

**Review by PrinceNiyonshuti — APPROVED** [review]
looks good

**Review by Xldivin — APPROVED** [review]

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]
Can you slightly fix these issues? The header should be responsive in your component as well

image

And this is how the landing page is looking in your branch, It's not the same case on what we have on develop branch

image

**Comment by rachelakimana** [comment]
@PatrickNiyogitare28 , you can review this PR again, I have done with the request changes

**Review by thepatrickniyo — APPROVED** [review]

**Pull request merged**